app/Http/Kernel.php

```diff
@@ -27,6 +27,7 @@ class Kernel extends HttpKernel {
 		'auth' => 'Gistlog\Http\Middleware\Authenticate',
 		'auth.basic' => 'Illuminate\Auth\Middleware\AuthenticateWithBasicAuth',
 		'guest' => 'Gistlog\Http\Middleware\RedirectIfAuthenticated',
+        'csrf' => 'Gistlog\Http\Middleware\VerifyCsrfToken',
```

Comment:
indentation issues but doesn't really bother me all that much since we're bout to re-indent the whole thing anyway